fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+swagger: '2.0'
+  
+info:
+  version: 1.0.0 
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_black.png"
+    backgroundColor:
+  
+  description: >
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the order is set to `completed` status. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
+      
+      * Create a draft order that calculates taxes and totals but accept payment from the customer outside of Shopify.
+      * Create a draft order (with taxes and totals) and send your customer a web checkout link to complete purchase and transition to a completed order.  
+      * Use Draft Orders API in conjunction with the [Checkouts API](https://help.shopify.com/api/tutorials/sell-through-the-checkout-api) by creating a Checkout object based on a draft order (Sales Channel SDK only).  This enables a highly customizable experience for channels.
+      
+     ## About Draft Orders
+    
+     When using the [Orders API](https://help.shopify.com/api/reference/order) to create orders, no payment information is calculated and no transaction is performed.  With the Draft Orders API, on the other hand, you can create orders on which shipping, taxes and all total amounts are calculated by Shopify.  This enables a wide range of use cases including the following:
+     
```

Comment:
we're missing a line of empty space between the intro paragraph and the bulleted list here:

![screen shot 2016-11-28 at 9 02 36 am](https://cloud.githubusercontent.com/assets/10617892/20670982/7ab2560e-b549-11e6-8e25-088df0930c3d.png)

Odd, because what appears to be the same formatting creates the desired spacing in the section above...

![screen shot 2016-11-28 at 9 03 34 am](https://cloud.githubusercontent.com/assets/10617892/20671014/97c6c252-b549-11e6-9702-365f7b76b1c9.png)

🤔